datasetinsights/stats/visualization/keypoints_plot.py

```diff
@@ -0,0 +1,102 @@
+""" Helper keypoints library to plot keypoint joints and skeletons  with a
+simple Python API.
+"""
+
+
+def get_color_from_color_node(color):
+    """ Gets the color from the color node in the template.
+
+    Args:
+        color (tuple): The color's channel values expressed in a range from 0..1
+
+    Returns: The color for the node.
+
+    """
+    r = int(color["r"] * 255)
+    g = int(color["g"] * 255)
+    b = int(color["b"] * 255)
+    a = int(color["a"] * 255)
+    return r, g, b, a
+
+
+def get_color_for_bone(bone):
+    """ Gets the color for the bone from the template.
+
+    Args:
+        bone: The active bone.
+
+    Returns: The color of the bone.
+
+    """
+    if "color" in bone:
+        return get_color_from_color_node(bone["color"])
+    else:
+        return 255, 0, 255, 255
+
+
+def get_color_for_keypoint(template, keypoint):
+    """ Gets the color for the keypoint from the template.
+
+    Args:
+        template: The active template.
```

Comment:
Can you add an example of the `template`? What's the structure?

Reply:
documentation updated